sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Generated/Models/Request.cs

```diff
@@ -24,7 +24,7 @@ public Request()
         /// <param name="properties"> Collection of custom properties. </param>
         /// <param name="name"> Name of the request, e.g., 'GET /values/{id}'. </param>
         /// <param name="url"> Request URL with all query string parameters. </param>
-        /// <param name="responseCode"> Result of a request execution. For http requestss, it could be some HTTP status code. </param>
+        /// <param name="responseCode"> Result of a request execution. For http requests, it could be some HTTP status code. </param>
```

Comment:
Don't modify the generated file, it will get overwritten with new updates. This comment should be fixed in swagger.